stdlib/src/builtin/int.mojo

```diff
@@ -692,14 +691,14 @@ struct Int(
         """
         if rhs == 0:
             # this should raise an exception.
-            return StaticIntTuple[2](0, 0)
+            return Tuple(0, 0)
         var div: Int = self._positive_div(rhs)
         if rhs > 0 and self > 0:
-            return StaticIntTuple[2](div, self._positive_rem(rhs))
+            return Tuple(div, self._positive_rem(rhs))
         var mod = self - div * rhs
         if ((rhs < 0) ^ (self < 0)) and mod:
-            return StaticIntTuple[2](div - 1, mod + rhs)
-        return StaticIntTuple[2](div, mod)
+            return Tuple[Int, Int](div - 1, mod + rhs)
```

Comment:
```suggestion
            return div - 1, mod + rhs
```
This should be enough. By default, values separated by commas means that it's a tuple. Same for the other returns